vtex/toolbelt
# Improve SSE errors

#### What is the purpose of this pull request?
This PR adds more info on `SSEErrors` so we can investigate further.
Also I did some minor refactorings.

#### How should this be manually tested?
```
cd /tmp && rm -rf toolbelt && \
git clone https://github.com/vtex/toolbelt.git && \
cd toolbelt && \
git checkout fix/sse-errors && \
yarn && yarn watch
```
To test if the SSE is still working properly:
- Try logging in.
- Try linking an app and check if logs are still reaching toolbelt.

#### Screenshots or example usage
SSEError example: https://splunk72.vtex.com/en-US/app/vtex_io_apps/search?earliest=-4h%40m&latest=now&q=search%20index%3Dio_vtex_logs%20app%3Dvtex.toolbelt-telemetry%40*%200542e715c4b40d48&display.page.search.mode=fast&dispatch.sample_ratio=1&display.general.type=events&sid=1587746004.644090_3DDE77BC-1901-4189-B912-ADE751E100F6&workload_pool=

#### Types of changes

#### Chores checklist
